_security-analytics/usage/overview.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: The Overview page
+parent: Using security analytics
+nav_order: 25
+---
+
+# The Overview page
+
+When you select **Security Analytics** from the top menu, the Overview page is displayed. The Overview page consists of five sections:
+* Findings and alert count.
+* Top recent alerts.
+* Top recent findings.
+* Most frequent detection rules.
+* Detectors.
+
+Each section provides a summary description for each element of security analytics, along with controls that let you take action for each item.
+
+## Overview and getting started
+
+The upper-right corner of the Overview page contains two control buttons for refreshing information and getting started with security analytics. You can select the **Refresh** button to refresh all of the information on the page. You can also select the **Getting started** link to expand the Get started with Security analytics window, which inludes a summary of the steps for set up as well as control buttons that allow you to jump to any of the steps.
```

Comment:
In the second sentence, should "Security" be capitalized? Change "as well as" to "and."

Reply:
This is a great question. Typically I capitalize a window name according to the way it appears in the UI. And this is the way it actually appears in the UI: Get started with Security analytics. But I'm making a command decision here and capitalizing Security Analytics. It still looks inconsistent. But I've decided to go back and capitalize Security Analytics everywhere. The inconsistency is driving me crazy, and I think the proper name of the plugin should be capitalized. Thanks.